openassessment/locale/en/LC_MESSAGES/djangojs.po

```diff
@@ -307,10 +307,6 @@ msgstr ""
 msgid "Thumbnail view of "
 msgstr ""
 
-#: openassessment/xblock/static/js/openassessment-lms.min.js:1
-msgid "Unable to load team detail"
-msgstr ""
```

Comment:
Any idea why this was removed from these two files?  It still seems to be in `djangojs.po` for all the other languages.  Rebasing might help, Transifex just updated all the translations 18 hours ago.

Reply:
I was not able to find this line in code which means this was removed from code. en is our base language so it is updated when we extract translations and eo is our dummy language its also updated locally. Once this is merged this will be pulled by transifex and will be removed from transifex. Other languages are updated when we pull new languages from transifex. we do not update them locally.